File: POO/PHP/Account.php
<?php
    class Account{
        public $id;
        public $name;
        public $document;
        public $email;
        public $password;
        
        public function __construct($name,$document){
            $this->name =$name;
            $this->document = $document;
        }

        public function toString(){
            return "I am a driver, my name is '{$this->name}' and my document is '{$this->document}'";
        }
    }
?>

File: POO/Java/Account.java
package POO.Java;

import java.util.UUID;

public class Account {
    UUID id;
    String name;
    String document;
    String email;
    String password;

    public Account(String name,String document, String email){
        id = UUID.randomUUID();
        this.name= name;
        this.document = document;
        this.email= email;
    }
}
